src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -120,16 +120,16 @@
         <button class="btn btn--red" on:click={showRemovePaymentCardDialog}>Remove</button>
       {/if}
 
-      <button class="btn-2 mrg-l mrg--l" on:click={() => showUpdatePaymentCardDialog()}>
+      <button class="btn-2 body-2" on:click={() => showUpdatePaymentCardDialog()}>
```

Comment:
`body-2` class breaks existing desktop UI. Please, make sure it's consistent with Figma design